news/733.feature

```diff
@@ -0,0 +1 @@
+Add endpoints for managing addons. [esteele]
```

Comment:
@rodfersou pls add your name here as well for credit. We can do this after the merge to not have to trigger another Jenkins job.

Reply:
will do

Reply:
Fantastic work @rodfersou. Thank you for making this happen!

Reply:
Fantastic work @rodfersou. Thank you for making this happen!

Reply:
I just did final touches, the fantastic work is your @esteele 😉